lib/dry/cli/command.rb

```diff
@@ -48,6 +49,10 @@ def self.extended(base)
         # @since 0.1.0
         # @api private
         attr_reader :options
+
+        # @since 0.6.0
```

Comment:
Please mark it like that, so that when I will prepare release it will be easy to find all occurrences `# @since 0.6.x`